### ISSUE TITLE
Double roundstart plasmaman loose stuff

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#33251]
(WEB REPORT BY: jesture REMOTE: 206.221.180.138:7777)
# Revision
4c4f1ff4843dc5759e848b469147fdc05dcf8204

# Description
getting double the loose plasmaman stuff when i joined as RD (exactly: RD coat, plasmaman tank, clipboard, plasmaman internals box, ID and PDA were doubled)
# Steps to Reproduce
>ready up as plasmeme RD (actually it was AI but i got filled RD)
>look at hands
>two clipboards
>wat
# What you Expected
normal amount of coats i can't wear
# What Actually Happened
forced to pawn off TWO drippy coats

**Comment by cloutcode** [comment]
tried this again but with RD as the highest prio and it didn't dupe stuff
maybe it has something to do with being reassigned the role or maybe it was just a strange case scenario
i checked the manifest and it didn't look like anyone else got a duped PDA

**Comment by pedr69** [comment]
yeah some funky stuff happens if your main job preference gets filled
#33015 probably related
